src/veil/ringct/lightwallet.cpp

```diff
@@ -0,0 +1,1890 @@
+//
+// Created by main on 25/2/22.
+//
+
+#include "lightwallet.h"
+#include <key.h>
+#include <key_io.h>
+#include <amount.h>
+#include <util/strencodings.h>
+#include <veil/ringct/watchonly.h>
+#include <veil/ringct/temprecipient.h>
+#include <secp256k1_mlsag.h>
+#include <veil/ringct/blind.h>
+#include <wallet/coincontrol.h>
+#include <veil/ringct/anonwallet.h>
+#include <policy/policy.h>
+#include <core_io.h>
+
+
+std::string TestBuildWalletTransaction(int nRandomInt) {
```

Comment:
This should be in a separate test file, under tests/ or something, not in the main binary.